test/Sentry.Tests/Internals/NetFxInstallationsEventProcessorTests.cs

```diff
@@ -0,0 +1,49 @@
+#if NETFX
+using Xunit;
+using Sentry.Internal;
+using System.Collections.Generic;
+using Sentry.PlatformAbstractions;
+
+namespace Sentry.Tests.Internals
+{
+    public class NetFxInstallationsEventProcessorTests
+    {
+        private class Fixture
+        {
+            public NetFxInstallationsEventProcessor GetSut() => new NetFxInstallationsEventProcessor();
+        }
+
+        private readonly Fixture _fixture = new Fixture();
+
+        [Fact]
+        public void Process_SentryEventWithNetFxList()
+        {
+            //Arrange
+            var @event = new SentryEvent();
+            var sut = _fixture.GetSut();
+
+            //Act
+            _ = sut.Process(@event);
+
+            //Assert
+            _ = Assert.IsAssignableFrom<IEnumerable<FrameworkInstallation>>(@event.Contexts[NetFxInstallationsEventProcessor.NetFxInstallationsKey]);
+        }
+
+        [Fact]
+        public void Process_NetFxInstallationsKeyExist_UnchangedSentryEvent()
```

Comment:
We need a test that verifies we can actually add something to the context. Maybe call `FrameworkInfo.GetInstallations()` and assert the items are there.

Ideally another that we can serialize the `contexts` once you add the items to that list.